client/src/assets/index.js

```diff
@@ -1,35 +1,35 @@
 /* eslint-disable import/prefer-default-export */
-import EVENT_1 from './events-page/past-events/22_23/1.svg';
-import EVENT_2 from './events-page/past-events/22_23/2.svg';
-import EVENT_3 from './events-page/past-events/22_23/3.svg';
-import EVENT_4 from './events-page/past-events/22_23/4.svg';
-import EVENT_5 from './events-page/past-events/22_23/5.svg';
-import EVENT_6 from './events-page/past-events/22_23/6.svg';
-import EVENT_7 from './events-page/past-events/22_23/7.svg';
-import EVENT_8 from './events-page/past-events/22_23/8.svg';
-import EVENT_9 from './events-page/past-events/22_23/9.svg';
-import EVENT_10 from './events-page/past-events/22_23/10.svg';
-import EVENT_11 from './events-page/past-events/22_23/11.svg';
-import EVENT_12 from './events-page/past-events/22_23/12.svg';
-import EVENT_13 from './events-page/past-events/22_23/13.svg';
-import EVENT_14 from './events-page/past-events/22_23/14.svg';
-import EVENT_15 from './events-page/past-events/23_24/1.svg';
+import EVENT_22_23_1 from './events-page/past-events/22_23/1.svg';
+import EVENT_22_23_2 from './events-page/past-events/22_23/2.svg';
+import EVENT_22_23_3 from './events-page/past-events/22_23/3.svg';
+import EVENT_22_23_4 from './events-page/past-events/22_23/4.svg';
+import EVENT_22_23_5 from './events-page/past-events/22_23/5.svg';
+import EVENT_22_23_6 from './events-page/past-events/22_23/6.svg';
+import EVENT_22_23_7 from './events-page/past-events/22_23/7.svg';
+import EVENT_22_23_8 from './events-page/past-events/22_23/8.svg';
+import EVENT_22_23_9 from './events-page/past-events/22_23/9.svg';
+import EVENT_22_23_10 from './events-page/past-events/22_23/10.svg';
+import EVENT_22_23_11 from './events-page/past-events/22_23/11.svg';
+import EVENT_22_23_12 from './events-page/past-events/22_23/12.svg';
+import EVENT_22_23_13 from './events-page/past-events/22_23/13.svg';
+import EVENT_22_23_14 from './events-page/past-events/22_23/14.svg';
+import EVENT_23_24_1 from './events-page/past-events/23_24/1.svg';
 
-export const EVENTS_23_24 = [EVENT_15];
+export const EVENTS_23_24 = [EVENT_23_24_1];
```

Comment:
Instead of doing `export const` twice, you can remove the `export` before the `const` and at the very bottom add `export {EVENTS_23_24, EVENTS_22_23}`